injected/integration-test/mocks/broker-protection/captcha.js

```diff
@@ -90,6 +102,18 @@ export function createSolveImageCaptchaAction(actionOverrides = {}) {
     });
 }
 
+/**
+ * @param {Partial<PirAction>} [actionOverrides]
+ */
+export function createSolveCloudFlareCaptchaAction(actionOverrides = {}) {
+    return createSolveCaptchaAction({
+        action: {
+            captchaType: 'image',
```

Comment:
shouldn't this say:

```suggestion
            captchaType: 'cloudFlareTurnstile',
```

?

Reply:
Credit to the captcha service detection since the tests continued to work. 😄